README.md

```diff
@@ -46,12 +46,128 @@ The `main` and `db` packages contain integration tests that require a successful
 ### No-scan domains
 In case of complaints or abuse, we may not want to continually scan some domains. You can set the environment variable `DOMAIN_BLACKLIST` to point to a file with a list of newline-separated domains. Attempting to scan those domains from the public-facing website will result in error codes.
 
-## API
+## Scan API
 
+Our API objects can look a bit complicated! There's lots of information contained in a TLS scan.
 To request a scan:
```

Comment:
Would it be useful to provide some information here about rate limits and blacklists so that people who might be tempted to use this for DoS attacks are dissuaded before they even start?

Reply:
If a scan isn't performed (or a new scan isn't performed) due to these limits, will the API just return older/empty scan result information, as opposed to specifically saying "you're not allowed to scan this host now" or something?